pom.xml

```diff
@@ -354,6 +354,9 @@ THE SOFTWARE.
           <groupId>org.apache.maven.plugins</groupId>
           <artifactId>maven-javadoc-plugin</artifactId>
           <version>2.10.3</version>
+          <configuration>
+            <quiet>true</quiet>
```

Comment:
Nope, I didn't. Fine for me if it still prints warnings/errors. Still unrelated to this PR, but not a bug then. Just 🐜